### PR TITLE
Map GraphQL scalar types to CLR types

### LINKED ISSUE #28: Viewer_By_GraphyQL_Matches_Api fails due to localization.
Running the `Viewer_By_GraphyQL_Matches_Api` integration test on my machine results in:

```
Xunit.Sdk.EqualException: Assert.Equal() Failure
Expected: 2016-05-04T23:20:55.0000000+00:00
Actual:   2016-04-06T01:20:55.0000000+02:00
   at Xunit.Assert.Equal[T](T expected, T actual, IEqualityComparer`1 comparer)
   at Xunit.Assert.Equal[T](T expected, T actual)
   at Octokit.GraphQL.IntegrationTests.Queries.ViewerTests.Viewer_By_GraphyQL_Matches_Api() in D:\projects\LinqToGraphQL\Octokit.GraphQL.IntegrationTests\Queries\ViewerTests.cs:line 57
```

I assume due to localization issues.

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
Fixes #28

Depends on:
- [x] #38

### CODE REVIEW

**Comment by StanleyGoldman** [comment]
Okay, so I spent some time with the project and I figured out why @iAmWillShepherd was having a hard time generating his new model.

Due to changes in the model, there was a list of other errors that needed to be solved, so we could get a baseline update that generated correctly. Only after that update in #38 is merged into this branch, can we generate a new model for this pull request.

File: Octokit.GraphQL.IntegrationTests/Utilities/Helper.cs [hunk 1]
```diff
@@ -13,9 +13,9 @@ public static class Helper
         public const string GithubComGraphqlApi = "https://api.github.com/graphql";
         public static readonly Uri GithubComApiUri = new Uri("https://api.github.com/");
 
-        public static DateTime ParseUtcLocal(string dateString)
+        public static DateTimeOffset ParseUtcLocal(string dateString)
         {
-            return Convert.ToDateTime(DateTime.Parse(dateString).ToLocalTime());
+            return DateTimeOffset.Parse(dateString).ToLocalTime();
```
Comment (grokys) [inline Octokit.GraphQL.IntegrationTests/Utilities/Helper.cs]: Using local time here is causing a test fail on my machine in `Viewer_By_GraphyQL_Matches_Api`:

```
Xunit.Sdk.EqualException: Assert.Equal() Failure
Expected: 2016-05-04T23:20:55.0000000+00:00
Actual:   2016-04-06T01:20:55.0000000+02:00
```

Why not just use the UTC time?

Comment (StanleyGoldman) [reply Octokit.GraphQL.IntegrationTests/Utilities/Helper.cs]: #20

Comment (StanleyGoldman) [reply Octokit.GraphQL.IntegrationTests/Utilities/Helper.cs]: #28

Comment (StanleyGoldman) [reply Octokit.GraphQL.IntegrationTests/Utilities/Helper.cs]: That function is actually no longer used.

Comment (StanleyGoldman) [reply Octokit.GraphQL.IntegrationTests/Utilities/Helper.cs]: And the code that validated the date was commented out.
Give it a shot now

Comment (StanleyGoldman) [reply Octokit.GraphQL.IntegrationTests/Utilities/Helper.cs]: Other tests were fixed too

**Pull request merged**